#this message will be used for motor sensor
ArmMotorInfo[] motorInfos
